suite-native/transactions/src/components/TransactionDetail/TransactionDetailData.tsx

```diff
@@ -60,13 +54,15 @@ export const TransactionDetailData = ({ transaction }: TransactionDetailDataProp
                     <TransactionDetailRow title="Fee">
                         <Box alignItems="flex-end">
                             <Text color="gray1000">
-                                {CryptoAmountFormatter.format(fee, {
+                                {CryptoAmountFormatter.format(transaction.fee, {
                                     symbol: transaction.symbol,
                                 })}
                             </Text>
                             <Text variant="hint" color="gray600">
-                                {`≈ ${FiatAmountFormatter.format(fiatFeeAmount ?? 0, {
```

Comment:
Prefer component version of formatters.

Reply:
This one I would leave as it is because of the symbol before and if it's in more places I would extend the formatter to do this. Then I would switch it

Reply:
Actually I think this `≈ <FiatAmountFormatter ... />` is more natural than template string.

Reply:
fixed